ShtefanIlya/HW5/hw_5.2.py

```diff
@@ -0,0 +1,35 @@
+# 0 1 1 2 3 5 8 13
+border = int(input("Enter a border:"))
```

Comment:
good job
but look at this implementation
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')